src/LibraryManager.Vsix/Shared/VsHelpers.cs

```diff
@@ -236,18 +234,29 @@ public static Project GetDTEProject(IVsHierarchy hierarchy)
 
             return null;
         }
-    }
 
-    public static class ProjectTypes
```

Comment:
Moving the ones we need to the Constants file. We had the wrong WAP GUID here. Fixed it at Constants file.